src/app/common/persistence.ts

```diff
@@ -17,6 +19,18 @@ const storage = {
 const chromeStorageLocalPersister = createAsyncStoragePersister({ storage });
 
 export const queryClient = new QueryClient({
+  queryCache: new QueryCache({
+    async onError(_error, query) {
+      const queryPrefix = query.queryKey[0] ?? '';
+      await Analytics().track(`query_error_${queryPrefix}`);
+    },
+  }),
+  mutationCache: new MutationCache({
+    async onError(_error, _variables, _context, mutation) {
+      const mutationPrefix = mutation?.options.mutationKey?.[0] ?? '';
+      await Analytics().track(`mutation_error_${mutationPrefix}`);
+    },
```

Comment:
For URL based errors, it's not critical we track current network state (can be inferred from url), and wallet type also not important. 

Maybe a `createAnalytics()` fn where we have a more basic one for use outside React, and the other one with more state, set in `useAnalytics`?

I almost wonder if we really need this at all. Let's try, but think we might be firing too many events.

Reply:
yep, that's fine, let's montiro